packages/blade/src/components/Link/BaseLink/BaseLink.tsx

```diff
@@ -255,6 +258,7 @@ const BaseLink = ({
       rel={rel ?? defaultRel}
       onClick={handleOnClick}
       disabled={disabled}
+      type={type}
```

Comment:
probably we can check if it's a link or a button and basis that just set this internally here?

Reply:
Do you mean checking if it's a link and not passing it in that case? 
A developer can pass these possible values, we can't infer the value in that case.
```ts
type?: 'button' | 'reset' | 'submit';
```

Reply:
Lets check if the consumer is intending to use it as a `button` variant and add `type='button'`. This will protect us from consumers passing `reset` or `submit`?

```tsx
      type={variant==='button'? 'button' : undefined}
```

Reply:
If we want to omit `reset` and `submit` for Link button variant, should I go ahead with making these changes to set `type='button'` based on variant?

Reply:
yes